src/components/StatusBadge/statusBadgeChangelogData.ts

```diff
@@ -9,6 +9,13 @@
 import { ChangelogData } from "../../utils/ComponentChangelogTable";
 
 export const changelogData: ChangelogData[] = [
+  {
+    date: "Prerelease",
+    version: "Prerelease",
+    type: "New Feature",
+    affects: ["Styles", "Functionality"],
+    notes: ["Adds the `labelFontSize` prop."],
```

Comment:
This is no longer true, but it say that the internal font size logic has been updated.